.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature request
+    url: https://techcommunity.microsoft.com/category/microsoft365/ideas/microsoft365developerplatform
+    about: To request a new feature in office.js or any other aspect of the Office developer platform, post your request on the Microsoft 365 Developer Platform Ideas Forum
```

Some suggestions to better align with how we talk about this [elsewhere](https://learn.microsoft.com/en-us/office/dev/add-ins/resources/resources-links-help#feature-request-and-general-help).
```suggestion
    about: To request a new feature in Office.js or the Office developer platform, please post your idea to the Microsoft 365 Developer Platform Tech Community.
```